scripts/etch-language-tests/maths/trig.etch

```diff
@@ -0,0 +1,112 @@
+function main()
```

Comment:
a variation on this already exists as an example. I recommend we delete the example since we don't need it if we have this test.